README.md

```diff
@@ -545,6 +545,16 @@ A pending promise can either be fulfilled with a value, or rejected with a reaso
 Promise![Promise](https://developer.mozilla.org/pl/docs/Web/JavaScript/Reference/Global_Objects/Promise)
 
 * What are the pros and cons of using Promises instead of callbacks?
+
+pros:
+  - avoiding "callback hell"
+  - more flexibility
+  - dealing with a proper data type, instead of a function
```

Comment:
Też mi to zdanie nijak nie pasowało, chodziło o coś w stylu, że mamy do czynienia z odpowiednim interfejsem danych zamiast zwykłych funkcji. Już poprawione.